enrich-points-with-boundaries/build.gradle

```diff
@@ -6,6 +6,12 @@ allprojects {
 	}
 }
 
+subprojects {
+	tasks.withType(Jar) {
+		archivesBaseName = "${project.parent.name}-${project.name}"
```

Comment:
My only thought was whether or not we might want to include the scala version in the artifact name.  It would be an added clue for those that don't look at the build file and just build it and try to deploy it.

Reply:
I like this suggestion, but could not find a clean way to do it without putting this line back in each subprojects gradle file.